dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -258,22 +259,48 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 					const ophanName = ophanComponentId(collection.displayName);
 					const ophanComponentLink = `container-${index} | ${ophanName}`;
+					const COTTON_CAPITAL_THRASHERS = [
```

Comment:
I'd love if there's something we can do to not need this - having one agreed upon solution for how these can work in DCR would definitely be the best of both worlds